<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="display_binary">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_maxThr_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>340</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>thr_max</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_maxThr_3">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>370</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>thr_min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_maxThr">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>340</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_minThr">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>370</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_min">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>380</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="display_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>640</width>
     <height>481</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_max">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>350</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_cam">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cam">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>20</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>60</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Recording Time (min.sec)</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_recording">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>80</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="plainText">
    <string>0.0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_startRecording">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>410</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Start recording</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>851</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_recordTime">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>490</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00:00 / 00:00:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_selectDirectory">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>190</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select directory to save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_saveDirectory">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>230</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_name">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>140</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>120</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cam</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_recordingStop">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>460</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <zorder>display_label</zorder>
  <zorder>display_binary</zorder>
  <zorder>label_maxThr_2</zorder>
  <zorder>label_maxThr_3</zorder>
  <zorder>label_maxThr</zorder>
  <zorder>label_minThr</zorder>
  <zorder>horizontalSlider_min</zorder>
  <zorder>horizontalSlider_max</zorder>
  <zorder>comboBox_cam</zorder>
  <zorder>pushButton_cam</zorder>
  <zorder>label</zorder>
  <zorder>plainTextEdit_recording</zorder>
  <zorder>pushButton_startRecording</zorder>
  <zorder>progressBar</zorder>
  <zorder>label_recordTime</zorder>
  <zorder>pushButton_selectDirectory</zorder>
  <zorder>label_saveDirectory</zorder>
  <zorder>plainTextEdit_name</zorder>
  <zorder>label_3</zorder>
  <zorder>label_2</zorder>
  <zorder>pushButton_recordingStop</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
